ses

```diff
@@ -330,3 +330,15 @@ fi
 plugin_command "systemctl status -l 'salt*'"
 
 #############################################################
+section_header "LIO iSCSI"
+
+if [ -d "/sys/kernel/config/target/" ]; then
+    find /sys/kernel/config/target/core/*/*/attrib \
+         /sys/kernel/config/target/iscsi/*/tpgt_*/attrib \
+         /sys/kernel/config/target/iscsi/*/tpgt_*/param \
+         -type f -printf '%p: ' -exec 'cat' '{}' ';'
```

Comment:
>   >  Does your target IQN have any images/LUNs associated with it?
> 
> No, it doesn't. My setup is perhaps a little broken though (somehow rbd -n client.iscsi.iscsi.node2.fpaakj device map -p iscsi myimg hangs when trying to attach an image to a target)

Hmm, sounds like a generic kRBD issue - can the client reach the mon. Is the image locked by another node?

Anyhow, to avoid the non-existent path errors, we could do the wildcard matching with find itself, instead of shell globbing e.g.
```console
# For LIO we're interesting in the following ConfigFS paths:
# (1) backstore device attribute files
# (2) iSCSI LUNs with their corresponding backstore device symlink
# (3) iSCSI transport layer attributes
# (4) iSCSI transport layer parameters
find /sys/kernel/config/target \
     -path "*/core/*/*/attrib/*" -type f \
     -printf '%p: ' -exec 'cat' '{}' ';' \
     -or -path "*/iscsi/*/lun/*" -type l \
     -printf '%p: %l\n' \
     -or -path "*/iscsi/*/tpgt_*/attrib/*" -type f \
     -printf '%p: ' -exec 'cat' '{}' ';' \
     -or -path "*/iscsi/*/tpgt_*/param/*" -type f \
     -printf '%p: ' -exec 'cat' '{}' ';'
```
This should also be a little more efficient compared to shell globbing.

> When I output this command, I get targets images from the core side, then iqn's from the iscsi path branch but can't tell which iqn points to what rbb image (I have 2 images).

Agreed, those links would be nice to have. I've added a lun symlink dump in the find command above, and will push as a v2.